test/native/setup.js

```diff
@@ -222,6 +222,12 @@ jest.mock( '@wordpress/compose', () => {
 			mockComponent( 'ResizeObserverMock' )( {} ),
 			{ width: 100, height: 100 },
 		] ),
+		usePreferredColorSchemeStyle: jest.fn( () => {
```

Comment:
These mock values are effectively "arbitrary," but were taken from values like `$light-quaternary` in `gutenberg/packages/base-styles/_colors.native.scss`